src/components/ControllerTop.vue

```diff
@@ -117,8 +116,13 @@ export default {
     isFavoriteKeyboard() {
       return this.keyboard === this.configuratorSettings.favoriteKeyboard;
     },
-    realKeymapName() {
-      return this.$store.getters['app/keymapName'];
+    keymapName: {
+      get() {
+        return this.$store.state.app.keymapName;
```

Comment:
this isn't the same.  the getter has additional functionality
```
  keymapName: (state) => {
    return state.keymapName.replace(/\s/g, '_').toLowerCase();
  },
```

Reply:
Ah, I didn't check that when I was creating PR.

After looking at the source code again I found that `setKeymapName` mutation looks like this:
```javascript
  setKeymapName(state, _keymapName) {
    state.keymapName = _keymapName.replace(/[\s/]/g, '_').toLowerCase();
  },
```
It looks like `keymapName` getter isn't necessary, because there's already value with underscores instead of whitespaces in store.

Reply:
I think this also changes as other things modify the keymap name. Try loading a keymap from JSON.

Reply:
I loaded file generated from config.qmk.fm with modified layout and "mateossh_test_layout" keymap name. 
No additional console messages, firmware did compile, downloaded files had proper names.
Then I manually set keymap name to "mateossh test layout with spaces" in file and tried to load it again in new tab.
Whitespaces were automatically changed to underscores, firmware did compile, files had proper names.

Reply:
Did UI change with the setting automatically?

Reply:
Yes